pandas/core/internals.py

```diff
@@ -2807,7 +2806,8 @@ def _astype(self, dtype, copy=False, errors='raise', values=None,
     def __len__(self):
         try:
             return self.sp_index.length
-        except:
+        except Exception:
```

Comment:
enumerate these

Reply:
Please read my previous comment.

Reply:
I did it was not helpful. you need to specify here.

Reply:
> I did it was not helpful. you need to specify here.

Of course it wasn't helpful.  It was asking for help!

Screw it, closing.

Reply:
> Any idea what the failure mode here is?

@jreback was the comment @jbrockmendel referred to. Saying it was "not helpful" is driving away contributors.

Reply:
@jbrockmendel my point is that saying ping I am ready when you haven't answer my questions is not helpful.

Reply:
I spend an enormous amounts of my time reviewing PR's and we have a lot of them. I do except answers to comments that I make.

Reply:
I'm guessing these last two comments are about a "ping" over in #19139, where I _had_ answered the question but that answer was ignored.